src/renderer/components/dialog-bisect.tsx

```diff
@@ -41,9 +41,11 @@ export const BisectDialog = observer(
       this.isEarliestItemDisabled = this.isEarliestItemDisabled.bind(this);
       this.isLatestItemDisabled = this.isLatestItemDisabled.bind(this);
 
+      const allVersions = this.props.appState.versionsToShow;
+
       this.state = {
-        allVersions: this.props.appState.versionsToShow,
-        startIndex: 10,
+        allVersions: allVersions,
+        startIndex: allVersions.length - 1 >= 10 ? 10 : allVersions.length - 1,
```

Comment:
```suggestion
        startIndex: allVersions.length > 10 ? 10 : allVersions.length - 1,
```

I think this makes it more clear what's going on here.